### DISCUSSION TITLE
I cannot find the code via the link "https://github.com/lu-group/fourier-mionet".

### DISCUSSION THREAD

**Discussion by liaoweiyang2017** [lululxvi/deepxde#1339] [Q&A]
I  have read your article "Fourier-MIONet: Fourier-enhanced multiple-input neural operators for multiphase modeling of geological carbon sequestration". And  I cannot find the code via the link you provide. Please update the code repository.

**Comment by lululxvi** [comment]
The code will be made public once the paper is accepted.
